### DISCUSSION TITLE
Inquiry Regarding Disabled Repository - PRABATH-MD

### DISCUSSION THREAD

**Discussion by prabathLK** [community/community#135763] [Repositories] [answered]
I recently discovered that the repository for our WhatsApp bot, PRABATH-MD, has been disabled. We’ve put a significant amount of effort into developing this project, and it’s disheartening to see it taken down. 

Could you please clarify the specific reason for the repository's disabling? We are eager to understand what went wrong so that we can address any issues and ensure compliance moving forward.

Here’s the link to the disabled repository: https://github.com/prabathLK/PRABATH-MD

Thank you for your assistance.

Best regards,  
prabathLK

**Comment by queenofcorgis** [comment, marked as answer]
Hi @prabathLK ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
